src/main/java/uk/gov/ons/census/action/schedule/ActionInstructionBuilder.java

```diff
@@ -110,6 +111,12 @@ public ActionInstruction buildPrinterActionInstruction(Case caze, ActionRule act
     actionAddress.setArid(caze.getArid());
     actionAddress.setUprn(caze.getUprn());
     actionAddress.setOa(caze.getOa());
+    if (caze.getLatitude() != null && !caze.getLatitude().isBlank()) {
```

Comment:
If lat/long are now considered mandatory, should we not fail here if either is `null` or blank, rather than just carrying on to send a message we know to be invalid?